src/NLog.Extensions.Logging/Extensions/ConfigureExtensions.cs

```diff
@@ -33,6 +35,19 @@ public static ILoggerFactory AddNLog(this ILoggerFactory factory, NLogProviderOp
             return factory;
         }
 
+        /// <summary>
+        /// Enable NLog as logging provider in .NET Core.
```

Comment:
not sure if this should be ".NET Standard." instead of ".NET Core"

Reply:
NetCore is the platform. NetStandard is the API-specification.

Reply:
But yes lets call it Net Standard since this packages works on both NetCore and NetFramework.

Reply:
Have updated all comments to say "for Microsoft Extension Logging".